python/examples/group_tracking.py

```diff
@@ -33,15 +32,21 @@ def ac_chat_modified(self, chat):
 
     @account_hookimpl
     def ac_member_added(self, chat, contact, actor, message):
-        print("ac_member_added {} to chat {} from {}".format(
-            contact.addr, chat.id, actor or message.get_sender_contact().addr))
+        print(
+            "ac_member_added {} to chat {} from {}".format(
+                contact.addr, chat.id, actor or message.get_sender_contact().addr
+            )
+        )
```

Comment:
In places like this one, the code got less nice. OTOH, not having to do the formatting yourself has advantages, too (not as big as in Rust code though because you still have to do the indentation correct, because the meaning of the code depends on it).

I guess that overall, I don't have an opinion on this.

Reply:
yes,  the whole idea is to avoid having to decide how a line like this looks "nice", bot are good enough, and actually black correction is better IMHO, at least to me it more clear that are nested parenthesis that are easier to match just from indentation, if there were 3 consecutive closing parenthesis `)))` things start to get hard to read